spec/lib/data_update_scripts/add_single_resource_admin_role_to_users_with_tech_admin_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20210203104631_add_single_resource_admin_role_to_users_with_tech_admin.rb",
+)
+
+describe DataUpdateScripts::AddSingleResourceAdminRoleToUsersWithTechAdmin do
+
+  before do
+    users = create_list(:user, 5)
```

Comment:
You don't need to assign this to a variable, it's never used. In fact, Rubocop should complain about this.

Reply:
It's weird that its not complaining. I also noticed that you fixed some rubocop stuff the other day in a PR related to the DUS, yet when I pushed the commit hook didn't alert. I'll double check if it's some setting on my side.

Reply:
Thanks for letting me know though, fixed  ✅

Reply:
Yes, I wondered about that recent fix PR too. Maybe there's some configuration issue on your side, let me know if you need any help figuring this out.